features/getscenesfromudcloud/README.md

```diff
@@ -0,0 +1,27 @@
+## Pulling Scenes and Files from a udCloud Project
+
+<!-- TODO: Write a brief abstract explaining this sample -->
+Feature sample to pull scenes and files from udCloud
+
+<!-- TODO: Fill this section below with metadata about this sample-->
+```
+Language:              (C-Style) C++
+Type:                  Feature Example
+Contributor:           Euclideon Development Team <support@euclideon.com>
+Organization:          Euclideon, https://euclideon.com
+Date:                  2020-08-03
+udSDK Version:         2.1.0
```

Comment:
2.2.1

Reply:
Actually with the breaking changes to udProject it'll be 2.3

Reply:
Fixed